volmaster/daemon.go

```diff
@@ -94,6 +82,7 @@ func (d *DaemonConfig) Daemon(debug bool, listen string) {
 	}
 
 	getRouter := map[string]func(http.ResponseWriter, *http.Request){
+		"/policy/{policy}":             d.handlePolicy,
```

Comment:
Let's put in an issue for this one. The whole hierarchy probably needs a  touch of rework.